src/storage/src/monitor/hummock_state_store_metrics.rs

```diff
@@ -341,6 +345,23 @@ impl HummockStateStoreMetrics {
             metric_level,
         );
 
+        // todo(wcy-fdu): may replace instance_id with actor_id.
```

Comment:
May use `metrics_level` if you want. (But current state LGTM)

Reply:
We may remove this metrics soon after mem table can spill.